checkpoint/cp-resource-management/README.md

```diff
@@ -1,85 +1,87 @@
-| `tolerations`                                              | List of node taints to tolerate                                 | `[]`                                             |
-| `affinity`                                                 | Affinity settings                                               | `{}`                                             |
-| `proxy`                                                    | Proxy settings (e.g. http://my-proxy.com:8080)                  | `{}`                                             |
+#  Check Point Cloudguard agents
+
+## Introduction
+
+This chart creates a single resource management Pod that scans the cluster's resources (Nodes, Images, Pods, Namespaces, Services, PSP, Network Policy, Role, ClusterRole, RoleBinding, ClusterRoleBinding, ServiceAccount, and Ingress) and uploads their meta-data to [Check Point ClougGuard](https://secure.dome9.com/).
+Check Point ClougGuard provides Posture Management, Visibility, Monitoring and Threat Hunting capabilities.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 3.0+
+- A Check Point ClougGuard account and API key
+
+## Installing the Chart
+
+To install the chart with the chosen release name (e.g. `my-release`), run:
+
+```bash
+$ helm repo add checkpoint https://raw.githubusercontent.com/CheckPointSW/charts/master/repository/
+$ helm install asset-mgmt checkpoint/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
+```
+
+This command deploys a CloudGuard Resource Management agent.
+
+> **Tip**: List all releases using `helm list`
+
+## Upgrading the chart
+
+To upgrade the deployment and/or to add/remove additional feature run:
+
+```bash
+$ helm repo update
+$ helm upgrade asset-mgmt checkpoint/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace]
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+This command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+In order to get the [Check Point ClougGuard](https://secure.dome9.com/) Cluster ID & credentials, you must first complete the Kubernetes Cluster onboarding process in [Check Point ClougGuard](https://secure.dome9.com/) website.
+
+Refer to [values.yaml](values.yaml) for the full run-down on defaults. These are a mixture of Kubernetes and CloudGuard directives that map to environment variables.
+
+Specify each parameter to `helm install` using `--set key=value[,key=value]` or `--set-string key=value[,key=value]`. For example,
```

Comment:
Specify each parameter using flags. 
For boolean parameters use the '--set' flag and for strings use the '--set-string' flag. 
For example: